### PR TITLE
Fix an issue with python3.8+

### PR DESCRIPTION
**Pull request by hadisfr**
due to deprecating `ast.Num`, `ast.Str`, `ast.Bytes`, `ast.Ellipsis` and `ast.NameConstant` in favour of `ast.Constant` in python3.8
ref: PyCQA/astroid#616
ref: https://bugs.python.org/issue32892#msg342486

Fix (**MAYBE PARTIALLY**) #17.

### CODE REVIEW

**Review by iradicek — CHANGES_REQUESTED** [review]
I'm not very happy with these generic ignore files. Do you mind putting explicitly what was causing the problems?

File: clara/py_parser.py [hunk 1]
```diff
@@ -74,6 +74,20 @@ def visit_Module(self, node):
 
     # Methods for visiting literals
 
+    def visit_Constant(self, node):
```
Comment (iradicek) [inline clara/py_parser.py:77]: Again, it would be cool to add some tests for this!

**Review by iradicek — CHANGES_REQUESTED** [review]

**Comment by hadisfr** [comment]
> I'm not very happy with these generic ignore files. Do you mind putting explicitly what was causing the problems? (This was supposed to be comment about `.gitignore`).

Actually it was just a boilerplate when I was trying to take a look at code and was not thinking about a merge request. I can remove that commit and do sth to other PRs.
Specially, I needed `.env` and `__pycache__/` to be ignored.

File: clara/py_parser.py (hunk 1 above)
Comment (hadisfr) [reply clara/py_parser.py:77]: I take a look at tests and try to add more. In some cases, like this one, it is not so straightforward, and may take me time to do that. Unfortunately I'm a little busy these days.

**Comment by iradicek** [comment]
> > I'm not very happy with these generic ignore files. Do you mind putting explicitly what was causing the problems? (This was supposed to be comment about `.gitignore`).
> 
> Actually it was just a boilerplate when I was trying to take a look at code and was not thinking about a merge request. I can remove that commit and do sth to other PRs.
> Specially, I needed `.env` and `__pycache__/` to be ignored.

Sure, then please do add those directories.

File: clara/py_parser.py (hunk 1 above)
Comment (iradicek) [reply clara/py_parser.py:77]: I'd add very simple tests that just parses & evaluates (using interpreter) given expressions.

If you're not able to do this in some time then I might finish these PRs once I have time.

Comment (hadisfr) [reply clara/py_parser.py:77]: It's very good if you can handle tests yourself. I've been far from code for a while.

**Comment by hadisfr** [comment]
> > > I'm not very happy with these generic ignore files. Do you mind putting explicitly what was causing the problems? (This was supposed to be comment about `.gitignore`).
> > 
> > 
> > Actually it was just a boilerplate when I was trying to take a look at code and was not thinking about a merge request. I can remove that commit and do sth to other PRs.
> > Specially, I needed `.env` and `__pycache__/` to be ignored.
> 
> Sure, then please do add those directories.

Done :D

File: clara/py_parser.py (hunk 1 above)
Comment (iradicek) [reply clara/py_parser.py:77]: Will add some tests #25.

**Review by iradicek — APPROVED** [review]

**Pull request merged**